src/hooks/useActionCenterSubscribers/useEvergreenDepositActionSubscriber.tsx

```diff
@@ -0,0 +1,79 @@
+import { usePrevious, useToast } from '@chakra-ui/react'
+import { fromAccountId } from '@shapeshiftoss/caip'
+import { TxStatus } from '@shapeshiftoss/unchained-client'
+import { useEffect } from 'react'
+
+import { useWallet } from '../useWallet/useWallet'
+
+import { useActionCenterContext } from '@/components/Layout/Header/ActionCenter/ActionCenterContext'
+import { EvergreenDepositNotification } from '@/components/Layout/Header/ActionCenter/components/Notifications/EvergreenDepositNotification'
+import { actionSlice } from '@/state/slices/actionSlice/actionSlice'
+import { selectPendingEvergreenDepositActions } from '@/state/slices/actionSlice/selectors'
+import { ActionStatus } from '@/state/slices/actionSlice/types'
+import { selectTxs } from '@/state/slices/selectors'
+import { serializeTxIndex } from '@/state/slices/txHistorySlice/utils'
+import { useAppDispatch, useAppSelector } from '@/state/store'
+
+export const useEvergreenDepositActionSubscriber = () => {
+  const { isDrawerOpen, openActionCenter } = useActionCenterContext()
+  const dispatch = useAppDispatch()
+  const {
+    state: { isConnected },
+  } = useWallet()
+
+  const toast = useToast({
+    duration: isDrawerOpen ? 5000 : null,
+    position: 'bottom-right',
+  })
+
+  const previousIsDrawerOpen = usePrevious(isDrawerOpen)
+
+  useEffect(() => {
+    if (isDrawerOpen && !previousIsDrawerOpen) {
+      toast.closeAll()
+    }
+  }, [isDrawerOpen, toast, previousIsDrawerOpen])
+
+  const pendingEvergreenActions = useAppSelector(selectPendingEvergreenDepositActions)
+  const txById = useAppSelector(selectTxs)
```

Comment:
Plural 
```suggestion
  const txs = useAppSelector(selectTxs)
```

Reply:
Going to stamp despite the blocking as this *does not* seem to be a perf bottleneck but still inherently bad for this to run so much

Reply:
Naming looking good now, removal of dep on txs still applicable.